cmake/s390x.cmake

```diff
@@ -0,0 +1,8 @@
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(SystemZ|s390x)")
+  message(STATUS "Forcing characters to be signed, as on x86_64.")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsigned-char")
+  message(STATUS "This platform does not support WAMR building so disabled.")
```

Comment:
Let's rephrase to, `Cannot build WAMR on this platform, so WASM support is disabled.`. Users won't care about WAMR (the implementation), they care whether or not they can use WASM.